frontend/public/components/dashboard/alerts-card/alerts-card.scss

```diff
@@ -0,0 +1,30 @@
+.co-alert__alerts-body {
+  max-height: 10em;
+  overflow-y: auto;
+  overflow-x: hidden;
+}
+
+.co-alert__item {
+  margin-bottom: 0.5em;
+  display: table;
+}
+
+.co-alert__item-message {
+  display: table-cell;
+}
+
+.co-alert__icon {
+  margin-right: 1.5em;
+}
+
+.co-alert__icon--critical {
+  color: #c9190b;
+}
+
+.co-alert__icon--warning {
+  color: #f0ab00;
+}
+
+.co-alert__top-margin {
+  margin-top: -15px;
```

Comment:
This is not a good way to do this.
If `AlertsCard` is not shown, it will cause spacing issues (empty `GridItem` is still rendered).
I'd just add alerts into `HealthCard`, so we wont have `AlertsCard`, but all will be part of `Health`

Reply:
Good point! Done!